con-master/src/main/java/ba/codecentric/MasterMain.java

```diff
@@ -0,0 +1,14 @@
+package ba.codecentric;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+/**
```

Comment:
Remove Default file template. It is ok if you want to add some additional information about class. But if there would be just a name of class creator, it is useless. One dev can create a class, but all other devs could work in it, so that info is not needed.